packages/clima_ui/lib/screens/settings_screen.dart

```diff
@@ -74,10 +74,7 @@ class SettingScreen extends ConsumerWidget {
                     return 'Imperial';
                 }
               }(),
-              leading: Icon(
-                Icons.straighten_outlined,
-                color: Theme.of(context).iconTheme.color,
-              ),
+              leading: const Icon(Icons.straighten_outlined),
```

Comment:
Are you sure that removing the color property didn't accidentally change the icon color to something else?

Reply:
From https://api.flutter.dev/flutter/widgets/Icon/color.html:

> Defaults to the current [IconTheme](https://api.flutter.dev/flutter/widgets/IconTheme-class.html) color, if any.

Feel free to run the app with all three themes to check for yourself. It's quite possible I made a blunder somewhere, but I think it's unlikely.